proto/README.md

```diff
@@ -29,28 +58,9 @@ transitioning from Tendermint Core to CometBFT.
 As of CometBFT v1, however, the project will transition to using and providing a
 `cometbft` package of Protobuf definitions (see [\#1330]).
 
-## How are `tendermint` Protobuf definitions versioned?
```

Comment:
Should we still keep links to the Buf registry ? I see there's a general link in the introduction at the top but maybe associating each version with the published version would be good.

Reply:
Restored the relevant text and the links in https://github.com/cometbft/cometbft/pull/1694/commits/c9fb60f915c9d486f979b5c734df622d38bf9123